app/Dockerfile.dev

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:focal
+
+COPY requirements.txt requirements.txt
+
+RUN apt-get	update && \
+	apt-get install -y --no-install-recommends gcc g++ build-essential apt-utils python3 python3-pip sqlite3 fftw3 python3-setuptools && \
+	apt-get install -y --no-install-recommends python3-dev libfftw3-dev git && \
+	pip3 install --no-cache-dir -r requirements.txt && \
+	apt-get remove -y python3-dev libfftw3-dev git && \
+	apt-get -y autoremove && \
+	rm -rf /var/lib/apt/lists/*
```

Comment:
Yes, I have added a volume for docker-compose.dev.yml with this Dockerfile.dev. So when we edit the host code it will changed in the container's code too. I'm not familiar with tornado whether it has hot-reloading feature.If it does, we can  use this to develop.

Reply:
I'm not sure what do you mean this copy from Dockerfile, do you mean lines 6~11?

Reply:
Yes the copy-pasted lines

Reply:
Thank you, I removed the Dockerfile.dev and use the same Dockerfile for both production and development docker-compose files.